.travis.yml

```diff
@@ -0,0 +1,58 @@
+sudo: false
+
+language: php
+
+notifications:
+  email:
+    on_success: never
+    on_failure: change
+
+branches:
+  only:
+    - /^v\d.*$/
+    - master
+    - develop
+
+php:
+  - 5.6
+  - 7.1
+
+env:
+  - WP_VERSION=4.4 WP_MULTISITE=0
+  - WP_VERSION=latest WP_MULTISITE=0
+  - WP_VERSION=trunk WP_MULTISITE=0
+
+matrix:
+  fast_finish: true
+  exclude:
+    - php: 7.1
+      env: WP_VERSION=4.4 WP_MULTISITE=0
+
+install:
+  - export DEV_LIB_PATH=dev-lib
+  - if [ ! -e "$DEV_LIB_PATH" ] && [ -L .travis.yml ]; then export DEV_LIB_PATH=$( dirname $( readlink .travis.yml ) ); fi
+  - if [ ! -e "$DEV_LIB_PATH" ]; then git clone https://github.com/xwp/wp-dev-lib.git $DEV_LIB_PATH; fi
+  - source $DEV_LIB_PATH/travis.install.sh
+
+script:
+  - source $DEV_LIB_PATH/travis.script.sh
+
+before_deploy:
+  - export RELEASE_FOLDER=madmimi-wp
```

Comment:
@EvanHerman The release folder name should match the slug on WordPress.org:

```bash
export RELEASE_FOLDER=madmimi-wp
```

https://wordpress.org/plugins/mad-mimi-sign-up-forms/